core/domain/exp_services.py

```diff
@@ -1408,7 +1870,16 @@ def create_or_update_draft(
     list timestamp of the new change list is greater than the change list
     timestamp of the draft.
     The method assumes that a ExplorationUserDataModel object exists for the
-    given user and exploration."""
+    given user and exploration.
+
+    Args:
+        exp_id: str. ID of the exploration.
+        user_id: str. ID of the user.
+        change_list: list(str). list containing the changes to made to the
+            ExplorationUserDataModel object.
+        exp_version: str. Version of the exploration domain object.
+        current_datetime: datetime.datetime. Present date and time. 
```

Comment:
The current date and time.

(Nitpicking here, but try not to introduce new words for the same thing if possible.)